Tests/GRPCTests/GRPCReflectionServiceTests/ReflectionServiceIntegrationTests.swift

```diff
@@ -33,9 +33,28 @@ final class ReflectionServiceIntegrationTests: GRPCTestCase {
   )
 
   private func setUpServerAndChannel() throws {
-    let reflectionServiceProvider = try ReflectionService(
-      fileDescriptors: self.protos + [self.independentProto]
-    )
+    var filePaths: [String] = []
```

Comment:
We actually don't need to do this in the integration tests: we can do it in the unit tests because the actual work is done by the registry rather than the service. This also means we can do this in a separate test as well.

Reply:
Oh, my mistake. Thinking about this some more, I think we can implement this as a couple of helpers on the `ReflectionService` which would be called by the `init`:

```swift
extension ReflectionService {
  static func readSerializedFileDescriptorProto(atPath path: String) throws -> Google_Protobuf_FileDescriptorProto { 
    // ...
  }

  static func readSerializedFileDescriptorProtos(atPaths paths: [String]) throws -> [Google_Protobuf_FileDescriptorProto] {     
    // ... 
  }
}
```

This way we can test the loading independently of the registry and operation of the service. I.e. we can test that these functions can load and deserialize the serialized descriptors correctly.